### ISSUE TITLE
Problems with random shut downs

### ISSUE THREAD

**Issue by Kitzoon** [ChrisTitusTech/win10script#326]
I enabled hibernation with windows toolbox and now my laptop is randomly shutting off at points while I use it.

Is this a problem just with hibernation? I think there should be a separate button to disable hibernation in windows toolbox to fix stuff like this if it is something to do with hibernation.

Note: My laptop goes into sleep, not actually fully shutting down but its REALLY annoying.

**Comment by Carterpersall** [comment]
If you ran essential tweaks, Hibernation will be disabled. If you ran essential tweaks undo, Hibernation will be enabled.
If you did in fact enable hibernation, try running `powercfg.exe /hibernate off` in PowerShell as admin. If the issue goes away, then the issue is with hibernation, and it should remain disabled. If the issue does not go away, then respond here and we can continue troubleshooting.

**Comment by Kitzoon** [comment]
sorry @Carterpersall I wasn't able to get on my laptop, I'll try the command you gave me in powershell and let you know if it solves the problem, thank you!

**Comment by Kitzoon** [comment]
still doing the same thing, even with me turning basically everything off sleep and shut down related in power settings and its still doing this

**Comment by Carterpersall** [comment]
@Kitzoon This probably isn't an issue with the script then. 
Try to make sure you don't bring anything magnetic close to your laptop to prevent it triggering it's close mechanism. I've had the issue that if I prop my phone up on my laptop it'll sometimes trigger it and go into sleep mode.
If that doesn't fix it, then try disabling fast startup if you haven't already.
If that doesn't work, you could try seeing if you change the lid-close action in Windows power settings from sleep to something else and see if the issue changes to that instead. If that changes the issue to what you set the action to, then it's probably a hardware issue causing your laptop to think it's being closed for some reason.
If none of these work, you could try reinstalling Windows to see if it's an issue with a corrupt version of Windows. But that should be the last thing you try.

**Comment by Kitzoon** [comment]
ill try to change the lid close action in my power options, thanks

**Comment by Kitzoon** [comment]
still turns off @Carterpersall

**Comment by Carterpersall** [comment]
Hmm theres still a few more things we can try before recommending a reinstall of Windows

 - Press Windows Key + X and click Powershell (Admin) for Windows 10 or Windows Terminal (Admin) for Windows 11 and run these commands and note the outputs of these commands:
      1: sfc /scannow
      2: DISM /Online /Cleanup-Image /RestoreHealth
      3: Restart and then run sfc /scannow one more time
   - This should attempt to remove corruption within Windows
   - This probably won't fix the issue, but it's always worth a try

 - If that doesn't fix your issue, you could try installing the program Caffeine (https://www.zhornsoftware.co.uk/caffeine/)
   - It claims to attempt to prevent Windows from going to sleep by simulating a button press about once a minute
   - If that doesn't work, you can also try Insomnia (https://dlaa.me/Insomnia/) which uses a different method to prevent the computer from going to sleep

**Comment by Kitzoon** [comment]
ive removed all the corruption windows can detect from those 2 commands, ill see if I get the issue now

**Comment by Kitzoon** [comment]
still get the problem

**Comment by Carterpersall** [comment]
At this point, the only thing I can think of that's left to do is a reinstall of Windows. It's more likely to be a hardware issue but it might be worth a shot.

**Comment by Kitzoon** [comment]
yeah to be honest im thinking of just switching to linux on my laptop then building a new pc or something

this laptop is prob on its death bed by now lol
